src/components/pagination/Pagination.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { typography, color, spacing } from "../../shared/style";
+import Icon from "../icon/Icon";
+
+const PaginationWrapper = styled.nav`
+  align-items: center;
+  border: 1px solid ${color.primary};
+  border-radius: ${spacing.borderRadius.medium}px;
+  display: flex;
+  font-weight: ${typography.weight.bold};
+  justify-content: center;
+  letter-spacing: 0.07rem;
+  vertical-align: top;
+
+  svg {
+    margin: ${spacing.padding.small}px;
+  }
+`;
+
+const Pagination = ({ currentPage, totalPages, onPrevPage, onNextPage }) => (
+  <PaginationWrapper>
+    <button
+      type="submit"
+      aria-label="previous page"
+      disabled={currentPage === 1}
+      onClick={onPrevPage}
+    >
+      <Icon aria-hidden="true" icon="sArrow" arrowDirection="left" />
```

Comment:
Wouldn't it be cleaner that way?
```suggestion
      <Icon 
      aria-hidden="true" 
      icon="sArrow" 
      arrowDirection="left" 
      />
```

Reply:
Maybe we are using different formatters. I use Prettier and the code looks like this after automatic formatting. Shouldn't be much of an issue :)

Reply:
Actually, this is kind of a big deal (not the lines here, but the code style in general 😊 ) I just realised I never set the prettier rules so that everyone would have the same formatting system and style. I will set up a `.prettierc` for this 😊